src/components/ReviewsView.jsx

```diff
@@ -0,0 +1,51 @@
+import { Card, Spinner } from "react-bootstrap";
+import { useState, useEffect } from "react";
+import * as api from "../api";
+
+const ReviewsView = () => {
+  const [allReviews, setAllReviews] = useState([]);
+  const [isLoading, setIsLoading] = useState(true);
+
+  useEffect(() => {
+    setIsLoading(true);
+    api.fetchReviews().then(({ data }) => {
+      console.log(data[0]);
+      setAllReviews(data);
+      setIsLoading(false);
+    });
+  }, []);
+
+  if (isLoading) return <Spinner animation="border" />;
+
+  return (
+    <div className="reviews">
+      {allReviews.map((review) => {
+        console.log(review);
+        return (
+          <Card className="card">
```

Comment:
This is getting a bit big, and it seems like all of the logic insider your Card tags is crying out to be separated out into it's own Card component file!